dot/network/connmgr_test.go

```diff
@@ -132,6 +129,10 @@ func TestProtectUnprotectPeer(t *testing.T) {
 }
 
 func TestPersistentPeers(t *testing.T) {
+	if testing.Short() {
+		t.Skip() // this sometimes fails on CI
```

Comment:
Maybe add a TODO with an issue number to fix the skipped tests? :thinking:

Reply:
yep I'll add it to here: #1026